xarray/conventions.py

```diff
@@ -543,8 +593,8 @@ def decode_cf(
         Decode cf times (e.g., integers since "hours since 2000-01-01") to
         np.datetime64.
     decode_coords : bool, optional
-        Use the 'coordinates' attribute on variable (or the dataset itself) to
-        identify coordinates.
+        Use the 'coordinates', 'grid_mapping', and 'bounds' attributes
+        on variables (or the dataset itself) to identify coordinates.
```

Comment:
```suggestion
        on variables (or the dataset itself) to identify coordinate variables.
```

Let's also link to the full list if we can.